octane-visual-studio-plugin-tests/octane-visual-studio-plugin-tests.csproj

```diff
@@ -38,23 +38,34 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.VisualStudio.TestPlatform.TestFramework, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\packages\MSTest.TestFramework.1.2.0\lib\net45\Microsoft.VisualStudio.TestPlatform.TestFramework.dll</HintPath>
-    </Reference>
-    <Reference Include="Microsoft.VisualStudio.TestPlatform.TestFramework.Extensions, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\packages\MSTest.TestFramework.1.2.0\lib\net45\Microsoft.VisualStudio.TestPlatform.TestFramework.Extensions.dll</HintPath>
-    </Reference>
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
     <Reference Include="System" />
     <Reference Include="System.Core" />
+    <Reference Include="System.Runtime.Serialization" />
+    <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="BaseOctanePluginTest.cs" />
+    <Compile Include="Common\FieldsCacheTests.cs" />
+    <Compile Include="Common\FieldsMetadataServiceTests.cs" />
     <Compile Include="Common\UtilityTests.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Utilities\ExposedClass.cs" />
+    <Compile Include="Utilities\ExposedObject.cs" />
+    <Compile Include="Utilities\ExposedObjectHelper.cs" />
+    <Compile Include="Utilities\Utility.cs" />
+    <Compile Include="ViewModel\DetailedItemViewModelTests.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\alm-octane-csharp-rest-sdk\OctaneSdkTests\OctaneSdkTest.csproj">
```

Comment:
Should the referenced solutions for debugging for development be commited? I mean if a someone check's out the code for the first time, it might be confusing

Reply:
This will go away once we have the new sdk nuget package